src/forms/NewStudyForm.js

```diff
@@ -1,351 +1,453 @@
-import React, {useState} from 'react';
+import React, {useState, Fragment} from 'react';
 import PropTypes from 'prop-types';
 import {
   Form,
   Segment,
   Container,
-  Label,
   Button,
   Message,
-  Grid,
   Confirm,
   List,
   Dropdown,
+  Step,
 } from 'semantic-ui-react';
 import {Formik} from 'formik';
 import {workflowOptions} from '../common/enums';
+import {PopupFormField} from '../components/PopupFormField';
 
-const NewStudyForm = ({submitValue, apiErrors}) => {
-  const [confirmOpen, setConfirmOpen] = useState(false);
-  const [workflowType, setSelection] = useState([
-    'bwa_mem',
-    'gatk_haplotypecaller',
-  ]);
+const InfoStep = ({
+  formikProps,
+  setActiveStep,
+  setFocused,
+  focused,
+  setSelection,
+  workflowType,
+}) => {
+  const {values, errors, touched, handleChange, handleBlur} = formikProps;
+  return (
+    <>
+      <Message info>
```

Comment:
I've been trying to reserve the `Message` component for notification only. Let's see what this looks like as just a heading tag